unit-tests/shared/src/test/scala/javalib/io/ByteArrayInputStream.scala

```diff
@@ -0,0 +1,38 @@
+package javalib.io
```

Comment:
This file name would be more consistent with the convention used for other file in `unit-test` if it were named `ByteArrayInputStreamTest.scala`.

Excellent to see a Test provided for introduced features.